lib/services/gitlab/gitlab_resource.rb

```diff
@@ -0,0 +1,44 @@
+class GitlabResource < GenericResource
+
+    def prepare_request
+        super
```

Comment:
Prefer two-space indentation -- the rest of the repo + new gitlab code uses two spaces per indent but this class uses four, should be updated to match.